### PR TITLE
Travis Fix

### PR DESCRIPTION
**Pull request by Dillon-Brown**
## Description

Increase wait time for addemailaccount test.

## How To Test This

See Travis

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

**Comment by cameronblaikie** [comment]
Assessed :+1:

File: tests/acceptance/modules/Users/UsersCest.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function testEmailSettingsMailAccountAdd(AcceptanceTester $I, UsersTester
         $I->fillField('email_user', 'testuser_name');
         $I->fillField('email_password', 'testuser_pass');
         $I->click('Test Settings');
-        $I->wait(10);
+        $I->wait(20);
```
Comment (connorshea) [inline tests/acceptance/modules/Users/UsersCest.php]: Is there no way to have the test suite wait until the AJAX call is finished (or the page is loaded, depending on what this is testing)?

Comment (Dillon-Brown) [reply tests/acceptance/modules/Users/UsersCest.php]: Good idea, should definitely be possible. Ideally I don't want a test that takes 20+ seconds.

Comment (connorshea) [reply tests/acceptance/modules/Users/UsersCest.php]: We had `sleep`s like this cause a lot of test slowdowns on a Rails app I worked on before, and I believe that was our solution in the end. Hopefully however SuiteCRM is integrating with selenium provides a way to do this :)

**Pull request merged**